src/pkcs1_5.c

```diff
@@ -624,6 +625,15 @@ int pkcs_v1_5_sign_man_ace_v1_5(struct image *image,
 	EVP_PKEY_free(privkey);
 	return ret;
 }
+#else
+int pkcs_v1_5_sign_man_ace_v1_5(struct image *image,
+			    struct fw_image_manifest_ace_v1_5 *man,
+			    void *ptr1, unsigned int size1, void *ptr2,
+			    unsigned int size2)
+{
+	return -EINVAL;
```

Comment:
@juimonen So IIUC, the code that should be here was removed in the previous patch becuase it was in the wrong place and caused FW to be signed twice ?
@mwasko @marcinszkudlinski it looks like we need the MTL code added here, the code is in the wrong place currently, so not new development just a code move and revalidate. Just checking incase you already have this fix ?

Reply:
I had an misunderstanding in my own ver25 ssl3 code (first patch), ace functions we're not implemented for ssl3 at all -> I needed to make quick guards and stub (second patch), because git HEAD was not even compiling in systems with ssl3. So just wanted to fix tgl fast, I need to check the ace code later.

Reply:
ok, lets merge this as correct SSL3 code and adapt MTL to follow the same flow when using SSL3.